cardigan/stories/components/Buttons/Buttons.stories.tsx

```diff
@@ -1,21 +1,11 @@
 import { useState } from 'react';
-import ButtonInline from '@weco/common/views/components/ButtonInline/ButtonInline';
 import ButtonSolid from '@weco/common/views/components/ButtonSolid/ButtonSolid';
 import DropdownButton from '@weco/common/views/components/DropdownButton/DropdownButton';
 import CheckboxRadio from '@weco/common/views/components/CheckboxRadio/CheckboxRadio';
 import Control from '@weco/common/views/components/Buttons/Control/Control';
 import ToolbarSegmentedControl from '@weco/common/views/components/ToolbarSegmentedControl/ToolbarSegmentedControl';
 import { chevron, digitalImage, eye, gridView } from '@weco/common/icons';
 
-const ButtonInlineTemplate = args => <ButtonInline {...args} />;
-export const buttonInline = ButtonInlineTemplate.bind({});
-buttonInline.args = {
-  disabled: false,
-  text: 'Click me',
-  isOnDark: false,
-};
-buttonInline.storyName = 'ButtonInline';
-
 const ButtonSolidTemplate = args => <ButtonSolid {...args} />;
 export const buttonSolid = ButtonSolidTemplate.bind({});
 buttonSolid.args = {
```

Comment:
Should these arguments be updated (there is `isBig`)?

Reply:
The checkboxes in that story also throw me errors, but that's probably unrelated 😄

Reply:
It's an unrelated issue, but it is a bit gross. The checkboxes in the story are purely illustrative, but it would be nice to find a way to stop TS shouting at us